packages/api/src/index.ts

```diff
@@ -16,10 +16,16 @@
 
 export { createClient, createThinClient, isOk } from "./client";
 export type { Client, ObjectSet, ResultOrError, ThinClient } from "./client";
+export { default as createOpenApiRequest } from "./client/internal/net/createOpenApiRequest";
+export type {
+  BaseGroupValue,
+  Bucketing,
+  GroupValue,
+} from "./client/internal/net/types";
```

Comment:
Are we good exporting these from within the `internal` folder?

Reply:
I'd rather we move them if they are being exported. As evidenced by the generated SDK, its can be quite difficult to realize what things have become part of public API.

Reply:
will do in a flup